app/controllers/reviews_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
     if review.save
       draft.update(status: :changes_requested)
       draft.application.update(status: :changes_requested)
-      ReviewMailer.changes_requested(review).deliver_now
+      ReviewMailer.changes_requested_email(review).deliver_now
```

Comment:
I wasn't sure about the need to include user here because I didn't see it being used in the controller.